### ISSUE TITLE
Changing stream quality.

### ISSUE THREAD

**Issue by thomascscott** [marshallthornton/Twitch.tv-Plex-Plugin#2]
Is there a way to change the quality of the streams where available, as is able to be done when viewing streams on their website?

**Comment by hellojusi** [comment]
Any info on this? Would love to be able to watch in HD. Thanks!

**Comment by marshallthornton** [comment]
With the latest update Zakk made that was pulled in it should use the Justin.tv code to pull the stream itself which gives up to 720p from what I can tell. This is the best quality that is offered by the non-flash streaming API they provide.

It should auto-select the quality based on available bandwidth so you may notice the quality adjust as it detects more/less available bandwidth for the streaming.

**Comment by pencil** [comment]
I only get 480p through this plugin. Would be nice to have an option to force 720p.

**Comment by craigmccaskill** [comment]
Yeah, as of now there doesn't seem to be a way to view streams in anything other than 480p. Would love to force a higher bandwidth rate.

**Comment by ghost** [comment]
Any update on this? 480p streams on a tv look pretty horrible...

**Comment by SeanRoberts** [comment]
Would love to get high def streaming, especially since PlexConnect + Twitch is the only way to watch Twitch on Apple TV
